src/sentry/api/endpoints/auth_index.py

```diff
@@ -257,6 +257,15 @@ def put(self, request: Request):
 
         if request.user.is_superuser and request.data.get("isSuperuserModal"):
             request.superuser.set_logged_in(request.user)
+            logging.info(
```

Comment:
@Milkshak3s Would it not be better to move this logging into the `request.superuser.set_logged_in()` function in https://github.com/getsentry/sentry/blob/ea9e2e29653c5eb60741d68a1220657e24609eeb/src/sentry/auth/superuser.py#L328?

Reply:
Ah! Looks like this logging case is already covered in that function:

https://github.com/getsentry/sentry/blob/ea9e2e29653c5eb60741d68a1220657e24609eeb/src/sentry/auth/superuser.py#L387-L394

I'll close this out. Thanks @dashed !